WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -144,16 +144,9 @@ class MainActivity : AppCompatActivity() {
         }
 
         // Configure asset loader with custom domain
-        // * NOTE THAT *:
-        // The assets path handler is set with the sub path /Asset-Loader/ here because we are tyring to ensure
-        // that the address loaded with loadUrl("https://gcoleman799.github.io/Asset-Loader/assets/index.html") does
-        // not conflict with a real web address. In this case, if the path were only /assests/ we would need to load
-        // "https://gcoleman799.github.io/assets/index.html" in order to access our local index.html file.
-        // However we cannot guarantee "https://gcoleman799.github.io/assets/index.html" is not a valid web address.
-        // Therefore we must let the AssetLoader know to expect the /Asset-Loader/ sub path as well as the /assets/.
         val assetLoader = WebViewAssetLoader.Builder()
-            .setDomain("gcoleman799.github.io")
-            .addPathHandler("/Asset-Loader/assets/", WebViewAssetLoader.AssetsPathHandler(this))
+            .setDomain("raw.githubusercontent.com")
```

Comment:
We should probably list these under the "views-widgets-samples" path to make sure there's no conflicts.